src/test/java/aquality/selenium/waitings/ConditionalWaitTests.java

```diff
@@ -102,71 +101,122 @@ public void testTimeoutExceptionShouldBeThrownIfDriverConditionIsNotMetAndTimeou
                         timer.start();
                         return false;
                     }, waitForTimeoutCondition, waitForTimeoutPolling,
-                    "Conditional should be true", Collections.singleton(StaleElementReferenceException.class));
+                    "Conditional should be true");
 
         } catch (org.openqa.selenium.TimeoutException e) {
             DurationSample durationSample = new DurationSample(timer.duration(), waitForTimeoutCondition, defaultDeviation);
             assertTrue(durationSample.isDurationBetweenLimits(), durationSample.toString());
-        } finally {
-            BrowserManager.getBrowser().quit();
         }
     }
 
     @Test
-    public void testTimeoutExceptionShouldNotBeThrownIfDriverConditionIsMetAndDefaultTimeoutIsNotOver() {
+    public void testTimeoutExceptionShouldBeThrownIfDriverConditionIsNotMetAndTimeoutIsOverWithIgnoredExceptions() {
         Timer timer = new Timer();
         try {
             ConditionalWait.waitFor((driver) ->
                     {
                         timer.start();
-                        return true;
-                    },
-                    "Conditional should be true");
-            DurationSample durationSample = new DurationSample(timer.duration(), getTimeoutConfig().getCondition());
-            assertTrue(durationSample.getDuration() < getTimeoutConfig().getCondition());
-        } finally {
-            BrowserManager.getBrowser().quit();
+                        return false;
+                    }, waitForTimeoutCondition, waitForTimeoutPolling,
+                    "Conditional should be true", Collections.emptyList());
+
+        } catch (org.openqa.selenium.TimeoutException e) {
```

Comment:
please import the type

Reply:
I did so, because there is already imported java.util.concurrent.TimeoutException;